oximeter/db/src/native/io/column.rs

```diff
@@ -378,7 +385,30 @@ fn encode_value_array(
                 io::string::encode(value, &mut dst);
             }
         }
-        ValueArray::Uuid(values) => copyout_pod_values!(Uuid, values, dst),
+        ValueArray::Uuid(values) => {
```

Comment:
The most confusing part here was that the incorrect encoding we were using before is _reversible_. When you select UUIDs back out, they look correct! I only caught this because we had some tests that were selecting them as strings, where the byte-swapping was apparent.